packages/platform/types/bookings/2024-08-13/inputs/create-booking.input.ts

```diff
@@ -122,16 +123,20 @@ export class CreateBookingInput_2024_08_13 {
   @IsOptional()
   location?: string;
 
-  // todo(Lauris): expose after refactoring metadata https://app.campsite.co/cal/posts/zysq8w9rwm9c
-  // @ApiProperty({
-  //   type: Object,
-  //   description: "Optional metadata for the booking.",
-  //   example: { key: "value" },
-  //   required: false,
-  // })
-  // @IsObject()
-  // @IsOptional()
-  // metadata!: Record<string, unknown>;
+  @ApiProperty({
+    type: Object,
+    description:
+      "Not available for seated bookings - only for normal and recurring ones. You can store any additional data you want here. Metadata must have at most 50 keys, each key up to 40 characters, and string values up to 500 characters.",
```

Comment:
I see metadata being passed to createNewSeat, is this doc correct ?

Reply:
Well spotted the doc is outadated and seated bookings accept metadata. Fixed in [refactor: metadata doc](https://github.com/calcom/cal.com/pull/17708/commits/07cb4846853c58f27f4b3c994665c337daef0dff)